src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -963,129 +963,27 @@ FixedwingAttitudeControl::task_main()
-						roll_sp = (_manual.y * _parameters.man_roll_max)
-							  + _parameters.rollsp_offset_rad;
-					}
-
-					pitch_sp = _att_sp.pitch_body + _parameters.pitchsp_offset_rad;
-					throttle_sp = _att_sp.thrust;
-
-					/* reset integrals where needed */
-					if (_att_sp.roll_reset_integral) {
-						_roll_ctrl.reset_integrator();
-					}
-
-					if (_att_sp.pitch_reset_integral) {
-						_pitch_ctrl.reset_integrator();
-					}
-
-					if (_att_sp.yaw_reset_integral) {
-						_yaw_ctrl.reset_integrator();
-						_wheel_ctrl.reset_integrator();
-					}
+				roll_sp = _att_sp.roll_body;
+				pitch_sp = _att_sp.pitch_body;
+				yaw_sp = _att_sp.yaw_body;
+				throttle_sp = _att_sp.thrust;
 
-				} else if (_vcontrol_mode.flag_control_altitude_enabled) {
-					/*
-					 * Velocity should be controlled and manual is enabled.
-					*/
-					roll_sp = (_manual.y * _parameters.man_roll_max) + _parameters.rollsp_offset_rad;
-					pitch_sp = _att_sp.pitch_body + _parameters.pitchsp_offset_rad;
-					throttle_sp = _att_sp.thrust;
-
-					/* reset integrals where needed */
-					if (_att_sp.roll_reset_integral) {
-						_roll_ctrl.reset_integrator();
-					}
+				if (!_vcontrol_mode.flag_control_velocity_enabled) {
+					yaw_manual = _manual.r;
+				}
 
-					if (_att_sp.pitch_reset_integral) {
-						_pitch_ctrl.reset_integrator();
-					}
+				/* reset integrals where needed */
+				if (_att_sp.roll_reset_integral) {
+					_roll_ctrl.reset_integrator();
+				}
 
-					if (_att_sp.yaw_reset_integral) {
-						_yaw_ctrl.reset_integrator();
-						_wheel_ctrl.reset_integrator();
-					}
+				if (_att_sp.pitch_reset_integral) {
+					_pitch_ctrl.reset_integrator();
+				}
 
-				} else {
-					/*
-					 * Scale down roll and pitch as the setpoints are radians
-					 * and a typical remote can only do around 45 degrees, the mapping is
-					 * -1..+1 to -man_roll_max rad..+man_roll_max rad (equivalent for pitch)
-					 *
-					 * With this mapping the stick angle is a 1:1 representation of
-					 * the commanded attitude.
-					 *
-					 * The trim gets subtracted here from the manual setpoint to get
-					 * the intended attitude setpoint. Later, after the rate control step the
-					 * trim is added again to the control signal.
-					 */
-					roll_sp = (_manual.y * _parameters.man_roll_max) + _parameters.rollsp_offset_rad;
-					pitch_sp = -(_manual.x * _parameters.man_pitch_max) + _parameters.pitchsp_offset_rad;
-					/* allow manual control of rudder deflection */
-					yaw_manual = _manual.r;
-					throttle_sp = _manual.z;
-
-					/*
-					 * in manual mode no external source should / does emit attitude setpoints.
-					 * emit the manual setpoint here to allow attitude controller tuning
-					 * in attitude control mode.
-					 */
-					struct vehicle_attitude_setpoint_s att_sp = {};
-					att_sp.timestamp = hrt_absolute_time();
-					att_sp.roll_body = roll_sp;
-					att_sp.pitch_body = pitch_sp;
-					att_sp.yaw_body = 0.0f - _parameters.trim_yaw;
```

Comment:
@AndreasAntener I did not consider this in the new code. Doesn't seem right to me.